app/build.gradle

```diff
@@ -18,7 +18,13 @@ ext {
     PERMISSION_DISPATCHER_VERSION = "2.1.3"
     MULTIDEX_VERSION = "1.0.0"
     MAP_UTILS_VERSION = "0.4.3"
-    FACEBOOK_ANDROID_SDK_VERSION ="4.14.1"
+    FACEBOOK_ANDROID_SDK_VERSION = "4.14.1"
+
+    PAGERSLIDINGTABSTRIP = "1.0.1"
```

Comment:
you don't have to redefine these version, all of them should be tied to APPCOMPAT_VERSION